src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/HostingAbstractionsHostExtensions.cs

```diff
@@ -51,7 +51,8 @@ public static void Run(this IHost host)
         }
 
         /// <summary>
-        /// Runs an application and returns a Task that only completes when the token is triggered or shutdown is triggered.
+        /// Runs an application and returns a <see cref="Task"/> that only completes when the token is triggered or shutdown is triggered.
+        /// The <paramref name="host"/> instance is disposed of after running.
```

Comment:
Would it make sense to go update `ConfigureWebHostDefaults` API docs? (either instead or as well)

https://github.com/dotnet/aspnetcore/blob/52eff90fbcfca39b7eb58baad597df6a99a542b0/src/DefaultBuilder/src/GenericHostBuilderExtensions.cs#L15-L30

Reply:
See https://github.com/dotnet/aspnetcore/pull/32025